<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1591</width>
    <height>1106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BinaryTree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>1221</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Visualizer"/>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>123</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Y">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}
</string>
        </property>
        <property name="text">
         <string>Create Head</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}
</string>
        </property>
        <property name="text">
         <string>Create Leaf Node</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}
</string>
        </property>
        <property name="text">
         <string>Delete Node</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}
</string>
        </property>
        <property name="text">
         <string>Search Node</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Guardar">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
        </property>
        <property name="text">
         <string>Save Tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
        </property>
        <property name="text">
         <string>Open Tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showRecorridosButton">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
        </property>
        <property name="text">
         <string>Orders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="limpiar">
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
        </property>
        <property name="text">
         <string>Clean Screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/otro - copia.png);
border-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveScreenshot">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>590</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Downloads/1200px-Picture_icon_BLACK.svg.png);

QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>1201</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>780</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
    </property>
    <property name="text">
     <string>ALTURA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>780</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
    </property>
    <property name="text">
     <string>PROFUNDIDAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>780</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(119, 193, 75);
}</string>
    </property>
    <property name="text">
     <string>BALANCE</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>frame_2</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1591</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionYEP"/>
    <addaction name="actionAbrir_Arbol"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConvert_to_Binary_disorder"/>
    <addaction name="actionConvert_to_BST"/>
    <addaction name="actionConvert_to_AVL"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTwilight_Sparkle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionYEP">
   <property name="text">
    <string>Guardar Arbol</string>
   </property>
  </action>
  <action name="actionAbrir_Arbol">
   <property name="text">
    <string>Abrir Arbol</string>
   </property>
  </action>
  <action name="actionConvert_to_Binary_disorder">
   <property name="text">
    <string>Convert to Binary (disorder)</string>
   </property>
  </action>
  <action name="actionConvert_to_BST">
   <property name="text">
    <string>Convert to BST</string>
   </property>
  </action>
  <action name="actionConvert_to_AVL">
   <property name="text">
    <string>Convert to AVL</string>
   </property>
  </action>
  <action name="actionTwilight_Sparkle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Twilight Sparkle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
